### DISCUSSION TITLE
카카오 테크 부트캠프 1기 합격, 그리고 포기

### DISCUSSION THREAD

**Discussion by 1jeongg** [BDD-CLUB/challenge-every-week-posting#576] [5. 회고]
(구) 카카오 클라우드 스쿨에 풀스택과 AI 과정을 합쳐서 새로운 부트캠프를 만들었길래 지원했습니다!

개인 사정으로 합격을 취소시켰지만 면접이나 결과 확인 과정이 인상적이어서 블로그를 작성해봤습니다~

~~어떻게 합격과 동시에 불합격~~

[카카오 테크 부트캠프 1기 합격 후기](https://1jeongg.github.io/kakao-tech-bootcamp/)

**Comment by shkisme** [comment]
ㅋㅋㅋㅋㅋ아니 합격이랑 불합격 동시에 메일 받고 엄청 당황스러웠을 것 같네요 🤣
오 부산은행 단기 인턴!! 파이팅입니닷~~!
